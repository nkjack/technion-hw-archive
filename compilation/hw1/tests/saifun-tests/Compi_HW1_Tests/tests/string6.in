'Hello \57orld!\r\nThis\tis\t\63oo\00006C, as a\C0always.'
"Hello \57orld!\r\nThis\tis\t\63oo\00006C, as a\C0always."
'From\000020lowest\20to\020highest\21\7E\07E\007E\0007E\00007E\7Q\n'
'This should be empty\3A\1F\7F\00003A'
'no match!"
