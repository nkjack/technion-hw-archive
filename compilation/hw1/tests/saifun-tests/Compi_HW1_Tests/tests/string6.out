1 STRING Hello World!
This	is	cool, as always.
2 STRING Hello \57orld!\r\nThis\tis\t\63oo\00006C, as a\C0always.
3 STRING From lowest to highest!~~~~~Q

4 STRING This should be empty::
Error unclosed string
